### DISCUSSION TITLE
How to fix Smplayer full screen animation from maximized window?

### DISCUSSION THREAD

**Discussion by tazihad** [smplayer-dev/smplayer#154] [Q&A]
This is bugging me for quite a long time.Smplayer is by far my favorite player. But its a issue bugging me. When smplayer is maximized to full screen it jumped. Doesn't follow animation rule like vlc. Is there any way I can fix this? 
OS: Manjaro KDE
Version: 21.1.0 (revision 9475)
Using Qt 5.15.2 (compiled with Qt 5.15.2)
Using MPV 0.33.1-dirty

https://streamable.com/6mrate

**Comment by smplayer-dev** [comment]
I guess that animation is done by KDE. There'll probably be a way to turn it off.

**Reply by tazihad** [reply]
I only face this issue in smplayer.

**Reply by smplayer-dev** [reply]
There's no animation for fullscreen implemented in smplayer. This is done either by KDE or maybe Qt.

**Comment by tazihad** [comment]
Hello. I am bumping this issue again. This same animation bug is also happening in Ubuntu 20.04. I think it's not KDE special bug. Hope dev will look into it.

**Comment by smplayer-dev** [comment]
I think I can't fix this. The animation is not done by smplayer. I've just tested on Ubuntu 20.04 and yes, there's a sort of animation. I tested on KDE Neon and there's not animation at all. There's also an animation on Mac OS but it can disabled in the system preferences.

The animations can be turn off on Ubuntu with this command:
`gsettings set org.gnome.desktop.interface enable-animations false`

https://fostips.com/disable-animations-speed-up-gnome-ubuntu/
